articles/stereotypes/how-to-use-stereotypes/how-to-use-stereotypes.md

```diff
@@ -3,18 +3,16 @@ uid: stereotypes.how-to-use-stereotypes
 ---
 # How to use Stereotypes
 
-This how-to will explain how to define and use [Stereotypes](xref:stereotypes.about-stereotypes) in [Modules](xref:modules.about-modules).
+This how-to will explain how to define [Stereotypes](xref:stereotypes.about-stereotypes) that are contained in your [Modules](xref:modules.about-modules) and how to apply them to your applications.
 
-The Stereotype defined in the Module will are useable in two ways:
+A Stereotype defined in a Module are useable in the following ways:
 
-- By the Module author in the code of the module through the easy to use Api which is generated by the Module Builder when the software factory is run for it.
-- By users of the Module in Intent Architect who can apply it to elements.
```

Comment:
Stereotypes defined in Modules are useable in the following ways:

\- During use of the Designers in Intent Architect: Stereotypes can be manually or automatically applied to Elements by everyday users.
\- During module authoring in the IDE: Intent Architect generates "APIs" for Stereotypes for easily retrieving them and their properties for use in Templates and other aspects of module building.